packages/breadcrumb/src/index.vue

```diff
@@ -11,11 +11,7 @@
 
 <script lang="ts">
 import { defineComponent, provide, ref, onMounted } from 'vue'
-
-interface IBreadcrumbProps {
-  separator: string
-  separatorClass: string
-}
+import { IBreadcrumbProps } from './breadcrumb'
```

Comment:
Please use `import type { IBreadcrumbProps }`

Reply:
Fixed.

Reply:
Next time you can resolve this conversation after you addressed the comment ;)

Reply:
Ok.